### DISCUSSION TITLE
Upgraded from 1.0 to 1.4.1, Various Issues

### DISCUSSION THREAD

**Discussion by rdp1070** [dialogic-godot/dialogic#849] [Q&A]
Hey, 
I installed/started this project a long time ago, but now I have just installed the new version of Dialogic, and now it is broken.
When I try to open an old timeline to edit/look at it, clicking on it the Dialogic tab prints this error to the console: 

- res://addons/dialogic/Editor/TimelineEditor/TimelineEditor.gd:260 - Invalid get index 'version_string' (on base: 'Nil').
- res://addons/dialogic/Editor/TimelineEditor/TimelineEditor.gd:277 - Invalid type in function 'set_timeline' in base 'GDScript'. Cannot convert argument 1 from Nil to Dictionary

And as long as I have the timeline selected it keeps printing it, about 1-3 a second until I select something else. 

There is also various parts of the Theme section that are broken, and when I try to add a new theme (hoping to make one that has the new coding standard I imagine) it says:

-  res://addons/dialogic/Editor/MasterTree/MasterTree.gd:668 - Invalid call. Nonexistent function 'create_theme' in base 'ScrollContainer (ThemeEditor.gd)'.

Please Advise, 
Bobby

**Comment by rdp1070** [comment]
I think the json contained values that didn't make sense anymore, so the whole thing was busted.
Thankfully I wasn't that far into the project. I will keep an eye out if anything else looks busted but it seems fine now.

**Reply by Jowan-Spooner** [reply]
Are the errors still being printet? Keep in mind that until you restart (after an update), weird things are a bit expected.

**Reply by rdp1070** [reply]
Yeah after updating, I restarted it several times after it didn't work. 
So it seemed like because it was so old, and components and properties were in different places in the json file, it didn't work. 
It also wouldn't let me create a new one however, I think it was just in a bugged state. 
After deleting all of the json I had be storing things in, it began working again. Which is fine because I was at the start of my project anyway. 

Sorry my post caused confusion. I wasn't specific enough for an issue report, because I wasn't sure if I had messed up or it. 

I am not sure if there is like a json conversion that happens if you upgrade, or what, but that seems like it could be helpful.
